decidim-core/app/presenters/decidim/user_menu_presenter.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module Decidim
+  # A presenter to render menus
+  class UserMenuPresenter < MenuPresenter
+    def render
```

Comment:
This lacks documentation. Which kind of renderer is this? Also; can we make it more semantic? It looks like it just concatenates menu items, so it might be called `InlinePresenter` maybe?

Reply:
Also, I think we should add tests for this behavior.